diff -up wordpress/wp-includes/js/tinymce/plugins/media/plugin.js.rpm wordpress/wp-includes/js/tinymce/plugins/media/plugin.js
--- wordpress/wp-includes/js/tinymce/plugins/media/plugin.js.rpm	2014-08-22 01:53:18.000000000 +0200
+++ wordpress/wp-includes/js/tinymce/plugins/media/plugin.js	2014-09-08 13:10:15.717314458 +0200
@@ -44,10 +44,6 @@ tinymce.PluginManager.add('media', funct
 			return 'video/ogg';
 		}
 
-		if (url.indexOf('.swf') != -1) {
-			return 'application/x-shockwave-flash';
-		}
-
 		return '';
 	}
 
@@ -239,7 +235,7 @@ tinymce.PluginManager.add('media', funct
 		data.source1mime = guessMime(data.source1);
 		data.source2mime = guessMime(data.source2);
 		data.poster = editor.convertURL(data.poster, "poster");
-		data.flashPlayerUrl = editor.convertURL(url + '/moxieplayer.swf', "movie");
+		data.flashPlayerUrl = false;
 
 		tinymce.each(urlPatterns, function(pattern) {
 			var match, i, url;
